app/components/edit-analysis-details.js

```diff
@@ -127,18 +128,85 @@ export default Component.extend({
     });
   },
 
-  actions: {
 
-    async uploadFile(file) {
+
+  detailSaveUtil: task(function * (param){
+    let key=param;
+    yield this.get('detailSaveAjaxCallUtil').perform(param)
+    .then(() => {
```

Comment:
Don't use then there as well use normal code. You can use normal try catch to catch errors.